x/feegrant/keeper/msg_server.go

```diff
@@ -37,12 +37,17 @@ func (k msgServer) GrantFeeAllowance(goCtx context.Context, msg *types.MsgGrantF
 	}
 
 	// Checking for duplicate entry
-	f := k.Keeper.GetFeeAllowance(ctx, granter, grantee)
+	f, _ := k.Keeper.GetFeeAllowance(ctx, granter, grantee)
```

Comment:
I am not sure if  it is relevant. I think , it makes sense to keep f variable inside if block cause it is not used in the code below.
```
if f, _ := k.Keeper.GetFeeAllowance(ctx, granter, grantee); f != nil {

}
```